.github/workflows/sonar-cloud-static-analysis.yml

```diff
@@ -42,8 +43,6 @@ jobs:
           echo "sonar.projectVersion=${{ github.sha }}" >> ${{ github.workspace }}/sonar-project.properties
           echo "sonar.organization=serenityos" >> ${{ github.workspace }}/sonar-project.properties
           echo "sonar.cfamily.compile-commands=${{ github.workspace }}/Build/${{ env.SONAR_ANALYSIS_ARCH }}/compile_commands.json" >> ${{ github.workspace }}/sonar-project.properties
-          echo "sonar.cfamily.threads=2" >> ${{ github.workspace }}/sonar-project.properties
-          echo "sonar.cfamily.cache.enabled=false" >> ${{ github.workspace }}/sonar-project.properties
```

Comment:
I'm not sure how this is removing unnecessary config? We specifically are disabling the cache, but the default you mentioned has a cache already set to some default. I see that the docs mention that the key we are setting is deprecated, but I'm not sure how to get the equivalent behavior with the new properties.

I'm not sure what reasons @bgianfo had at the time for disabling it, but I'm assuming it's related to the fact that we make so many commits a day that the cache wouldn't be particularly useful when run in the nightly job here.

Reply:
According to the commit message, the cache was explicitly disabled to avoid having a warning: https://github.com/SerenityOS/serenity/commit/ca7bb812f65445f31c96a8b6168d9bae9ccb24b5

In the past, SonarCloud didn't have an automatic server caching mechanism, and the filesystem cache needed to be manually set up.
Now that the automatic server cache is available on SonarCloud we invite users to profit from it. It cannot make things worse, it can only improve analysis speed.

Reply:
I see, thanks! If it'll only help and not cause a warning then let's try it.